chat-commands.js

```diff
@@ -2640,6 +2641,9 @@ const commands = {
 		if (!reason) {
 			return this.errorReply(`Battle bans require a reason.`);
 		}
+		if (!room.battle && (!reason.includes('battle-') && cmd !== 'forcebattleban')) {
```

Comment:
A replay link is not going to include `battle-`. :|

Reply:
https://replay.pokemonshowdown.com/doublesou-232753081

example replay link

Reply:
```suggestion
		if (!room.battle && (!reason.includes('.pokemonshowdown.com/') && cmd !== 'forcebattleban')) {
```

is probably how I'd do it. False positives aren't a huge problem; these _are_ mods.

Reply:
Yeah I can do that, testing with random battles definitely wasn’t the best idea, whoops. ``pokemonshowdown.com`` covers both replays and active battles as well so that works. Will change

Reply:
Also battlebans can be used by drivers, so I was trying to be as safe as possible